sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -375,12 +375,30 @@ public void testSetGlobal() throws Exception {
 
     @Test
     public void testSetSession() throws Exception {
-        printStatement("set some_setting = '1'");
-        printStatement("set some_setting = '1', other_setting = 2");
-        printStatement("set session some_setting = '1'");
-        printStatement("set session some_setting = '1', other_setting = 2");
-        printStatement("set session some_setting TO '1'");
-        printStatement("set session some_setting TO '1', other_setting TO 2");
+        printStatement("set some_setting = DEFAULT");
+        printStatement("set some_setting = 'value'");
+        printStatement("set some_setting = 1, '2', 'foo'");
+
+        printStatement("set some_setting TO DEFAULT");
+        printStatement("set some_setting TO 'value'");
+        printStatement("set some_setting TO 1, 2, 3");
+
+        printStatement("set session some_setting = DEFAULT");
+        printStatement("set session some_setting = 'value'");
+        printStatement("set session some_setting = 1, 2, 3");
+
+        printStatement("set session some_setting TO DEFAULT");
+        printStatement("set session some_setting TO 'value'");
+        printStatement("set session some_setting TO 1, 2, 3");
+
+        printStatement("set local some_setting = DEFAULT");
+        printStatement("set local some_setting = 'value'");
+        printStatement("set local some_setting = 1, 2, 3");
+
+        printStatement("set local some_setting TO DEFAULT");
+        printStatement("set local some_setting TO 'value'");
+        printStatement("set local some_setting TO 1, 2, 3");
+
```

Comment:
`printStatement("set local some_setting = 'on'"); ` -> passes
` printStatement("set local some_setting = on");  `-> fails

I need to update the parser